mlflow/models/evaluation/base.py

```diff
@@ -473,6 +475,7 @@ def evaluate(self, *, model, model_type, dataset, run_id, evaluator_config, **kw
         :param run_id: The ID of the MLflow Run to which to log results.
         :param evaluator_config: A dictionary of additional configurations for
                                  the evaluator.
+        :param custom_metrics: A list of custom metric functions
```

Comment:
Show what's required for the custom metric function signatures, and give an example on how it could be used (similar to what's discussed in the design discussion).

Reply:
I have more written more detailed documentation along with example in the function docstring of the mlflow.evaluate function here https://github.com/mlflow/mlflow/blob/7b26a88d51d59ece67aca3a8db6a0140baf65822/mlflow/models/evaluation/base.py#L801-L839